pkg/api/vlabs/validate.go

```diff
@@ -818,6 +823,14 @@ func validatePoolName(poolName string) error {
 	return nil
 }
 
+func validatePoolOSType(os OSType) error {
+	// Should we use a.IsLinux() and a.IsWindows() instead?
+	if os != Linux && os != Windows {
```

Comment:
@JamesEarle just realized `""` is also a default value... If the user leaves osType blank the apimodel is valid (it defaults to Linux). that's why unit tests are failing :)

Reply:
Regarding your comment, either way is fine. Feel free to do it this way and remove the comment.